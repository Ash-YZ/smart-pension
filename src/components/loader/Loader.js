import React from 'react'
import './styles.scss'

const Loader = _ => {
  return (
    <div className="loader-wrapper">
      <div className="loader">
        <div className="face">
          <div className="circle"></div>
        </div>
        <div className="face">
          <div className="circle"></div>
        </div>
      </div>
    </div>
  )
}

export default Loader