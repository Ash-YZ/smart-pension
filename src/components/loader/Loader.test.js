import React from 'react';
import { shallow } from 'enzyme'
import Loader from './Loader'

describe('Loader', () => {
  it('should render loader correctly ', () => {
    const component = shallow(<Loader />)
    expect(component).toMatchSnapshot()
  })
})