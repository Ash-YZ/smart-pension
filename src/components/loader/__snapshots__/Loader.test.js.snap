// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render loader correctly  1`] = `
<div
  className="loader-wrapper"
>
  <div
    className="loader"
  >
    <div
      className="face"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="face"
    >
      <div
        className="circle"
      />
    </div>
  </div>
</div>
`;
